apiclient/container_inspect.go

```diff
@@ -0,0 +1,748 @@
+/*
+Copyright 2019 Thomas Weber
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apiclient
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+// ContainerInspect data takes a name or ID of a container returns the inspection data
+func (c *APIClient) ContainerInspect(ctx context.Context, name string) (InspectContainerData, error) {
+
+	var inspectData InspectContainerData
+
+	res, err := c.Get(ctx, fmt.Sprintf("/%s/libpod/containers/%s/json", PODMAN_API_VERSION, name))
```

Comment:
Do we know if version is expected to change often? I'm tempted to just hardcode /v1.0.0/ in the base url so we don't have to use the constant everywhere

Reply:
The entire "version" thing is a bit unclear. Their reference has a "v1.0.0" in the examples, although one can find quite different "versions" when browsing blogs from early adopters. I also found that on some endpoints it's no difference to omit the version string.  

Let's remove the const, ok? It could even be an advantage to have fine grained control in each single API call.

Reply:
See latest commit, the const is gone. Instead each api call has a appropriate path.